### DISCUSSION TITLE
MacOS testing? what are some things I should look at?

### DISCUSSION THREAD

**Discussion by pengolod** [IrisShaders/Iris#260] [Q&A]
I'd like to contribute to the project by testing what I can with my macOS machine - are there any specific configurations I can test atm?

**Comment by coderbot16** [comment]
> I'd like to contribute to the project by testing what I can with my macOS machine - are there any specific configurations I can test atm?

At the moment there's a few things that need to be done before I have any expectation of Iris working at all on macOS, but testing would be welcome once I've done those things.

**Reply by pengolod** [reply]
Ok - I've been running the minimal ui PR branch w/ Sildur's Vibrant Lite and BSL with few problems, most other shaders haven't worked for me though. But I haven't run into the problems that you keep expecting with macOS 🤷‍♂️

**Reply by pengolod** [reply]
Oh! And XorDev's ominous shaders work for me for the most part - they could be bugging out for all I know, but they didn't crash or otherwise look broken. Some of the gamma or exposure was hard to see through, but I could also believe that was intended behavior of the shader.

**Reply by ChuckFaulker** [reply]
I got it up and running on MacOS with the m1 ARM chip

**Reply by rogerfan48** [reply]
> I got it up and running on MacOS with the m1 ARM chip

what did you do? I cannot run it(I use Macbook Air m1 ARM chip)

**Reply by rogerfan48** [reply]
i can run it without sodium, too. But with sodium, it crashed.

**Reply by rogerfan48** [reply]
and if i run sodium only(without Iris), it can work. so i guess the problem might be the compatibility
